apps/system-e2e/src/tests/islandis/service-portal/smoke/occupational-licenses.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { BrowserContext, expect, test } from '@playwright/test'
+import { icelandicAndNoPopupUrl, urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+import { label } from '../../../../support/i18n'
+import { m } from '@island.is/service-portal/core/messages'
+import { disableI18n } from '../../../../support/disablers'
+
+const homeUrl = `${urls.islandisBaseUrl}/minarsidur`
+test.use({ baseURL: urls.islandisBaseUrl })
+
+test.describe('Occupational licenses overview', () => {
+  let context: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    context = await session({
+      browser: browser,
+      storageState: 'service-portal-faereyjar.json',
+      homeUrl,
+      phoneNumber: '0102399',
+      idsLoginOn: true,
+    })
+  })
+
+  test.afterAll(async () => {
+    await context.close()
+  })
+
+  test('Has occupational licenses', async () => {
+    const page = await context.newPage()
+    await disableI18n(page)
+
+    await test.step(
+      'Overview cards are displayed and cards can navigate to detail',
+      async () => {
+        // Arrange
+        await page.goto(icelandicAndNoPopupUrl('/minarsidur/starfsleyfi'))
+
+        const hasHealthDirectorateLicense = page
+          .locator(`role=button[name="${label(m.view)}"]`)
+          .first()
+
+        // Act
+        await hasHealthDirectorateLicense.click()
+        const healthDirectorateTitle = page.getByText('Sálfræðingur')
+        const healthDirectorateIsValid = page.getByText('Í gildi')
```

Comment:
Why not use `label(...)`?

Reply:
Not an experienced tester, whats the difference if you care to elaborate?

Reply:
The reason I used `getByText` is because the value is coming from an API reponse and is not translated.

Reply:
I was suggesting `label` to get translations and prevent future breakage when the text changes in code but not here.

If it's not in our strings library, should it maybe be?